### PR TITLE
Jmmabanta/feature/website refactor integration

### PR DESCRIPTION
**Pull request by jmmabanta**
Added support for faking responses when using 'dummy' as an interface (`-I dummy`).

Luckily, I've found a way to modify the system path variable (temporarily) when running the website's communication script so that the imports are able to work without any modification!

I've also modified cli.py to mimic the [old version](https://github.com/AlbertaSat/ex2_ground_station_software/blob/e80e1b1478b2a30415fd62c4655be953a79d80f3/src/cli.py#L65) where--if given a list of dictionaries--would pretty print each one out. Apparently, the HOUSEKEEPING.GET_HK command can return multiple dictionaries as a list depending on the arguments (though I could be wrong and this behavior could've changed).

### CODE REVIEW

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

**Comment by Taylorr82** [comment]
I'd like to see the results of @tganley testing https://github.com/AlbertaSat/ex2_ground_station_software/pull/125 before we merge this. Is that alright?

**Comment by jmmabanta** [comment]
Yeah Id prefer that to as I'd like to get the housekeeping part right for the website

File: src/cli.py [hunk 1]
```diff
@@ -30,8 +30,14 @@ def run(self):
                 transactObj = self.interactive.getTransactionObject(inStr, self.networkManager)
                 ret = transactObj.execute()
                 print()
-                for key, value in ret.items():
-                    print("{} : {}".format(key, value))
+                # Housekeeping data can be a list of dicts
```
[the remark this answers is not in the document]
Comment (jmmabanta) [reply src/cli.py:33]: Ill wait for #125 to see the format of HK first

**Comment by jmmabanta** [comment]
Status on #125?

**Comment by Taylorr82** [comment]
> Status on #125?

Still no communication from the other teams

**Comment by Taylorr82** [comment]
#125 is merged now

**Comment by jmmabanta** [comment]
Sounds good!. Can you have a final pass looking over my code just in case before merging? (I believe the only big change was adding dummy housekeeping responses and having 'GET_INSTANT_HK' use the hkTransaction instead of the normal transaction)

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**